Documentation/guides/messages/README.md

```diff
@@ -61,10 +61,11 @@ ms.date: 01/24/2020
 + [XA0113](xa0113.md): Google Play requires that new applications and updates must use a TargetFrameworkVersion of v8.0 (API level 26) or above.
 + [XA0114](xa0114.md): Google Play requires that application updates must use a `$(TargetFrameworkVersion)` of v8.0 (API level 26) or above.
 + [XA0115](xa0115.md): Invalid value 'armeabi' in $(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties to remove the old value. If the properties page does not show an 'armeabi' checkbox, un-check and re-check one of the other ABIs and save the changes.
-+ [XA0116](xa0116.md): Unable to find `EmbeddedResource` of name `{ResourceName}`.
++ [XA0116](xa0116.md): Unable to find `EmbeddedResource` named `{ResourceName}`.
```

Comment:
I think this preserves the intended meaning. I adjusted it for familiarity to the localization team since I think "named" is a more common wording than "of name." But let me know if this changes the meaning.